wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
```

Comment:
I think we're ok on terminology.  Embedded means in another rails app, or what you are suggesting is a *deck app*.  Then you started calling the standard case, *app deck* which is hard for a dyslexic like me.

I guess I wasn't thinking about it quite right.  I was thinking because you create the card dependency in a migration, that reloading the schema is out, but the migration expands the schema and so it can load the schema that will have all the apps other tables and the card tables.

I still need a task for loading the data.  I want to put it in the decko namespace (vs. wagn tasks where we already have them).  Maybe initially we have the same tasks in wagn and decko.  Ideally, we won't actually depend on wagn in the deck app case.